components/airtable/new-modified-or-deleted-records.js

```diff
@@ -0,0 +1,111 @@
+const airtable = require('https://github.com/PipedreamHQ/pipedream/components/airtable/airtable.app.js');
+const moment = require('moment');
+const axios = require('axios');
+const Bottleneck = require('bottleneck');
+
+const limiter = new Bottleneck({
+  minTime: 200, // 5 requets per second
+});
+const axiosRateLimiter = limiter.wrap(axios);
+
+module.exports = {
+  name: 'New, Modified or Deleted Records',
+  version: '0.0.2',
```

Comment:
Maybe set to `0.0.1` for the initial release?

Reply:
Yes. Thanks for pointing that out. I have set it to 0.0.1.

Reply:
Yes, have been set  to 0.0.1